editoast/src/views/timetable/paced_train.rs

```diff
@@ -462,12 +457,17 @@ async fn simulation(
         })
         .await?;
 
+    let train_schedule = exception_key
+        .and_then(|key| paced_train.exceptions.iter().find(|e| e.key == key))
+        .map(|exception| paced_train.apply_exception(exception))
+        .unwrap_or_else(|| paced_train.into_first_occurrence());
```

Comment:
Same remark as above, don't we want to fail if the `exception_key` is present but cannot be found?